tiled/server/settings.py

```diff
@@ -1,82 +1,50 @@
-import collections
 import os
 import secrets
 from datetime import timedelta
 from functools import cache
 from typing import Any, List, Optional
 
-from pydantic_settings import BaseSettings
+from pydantic import Field
+from pydantic.dataclasses import dataclass
+from pydantic_settings import BaseSettings, SettingsConfigDict
 
-DatabaseSettings = collections.namedtuple(
-    "DatabaseSettings", "uri pool_size pool_pre_ping max_overflow"
-)
+
+# hashable cache key for use in tiled.authn_database.connection_pool
+@dataclass(unsafe_hash=True)
```

Comment:
I suspect we will not need this, once we merge the refactor that places the connection pool in `app.state` instead of a global dict keyed on this object.